CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.5.0] 2021-04-19
+
+### Added
+- Added support for reading textfiles (optionally compressed) in chunks - may add
```

Comment:
totally a nit here, but i feel like a changelog should only indicate _what_ changed not potential future changes?

Reply:
You're totally right, I'll remove that before merging